kernel/src/managers/dma/dma-dt.c.in

```diff
@@ -86,7 +86,7 @@ static const dma_meta_t streams[] = {
             .source = {{ stream_get_transfer_source(node) }},
             .dest = {{ stream_get_transfer_dest(node) }},
             {% if node["length"] -%}
-            .transfer_len = {{ "%08u"|format(node["length"]) }}UL,
+            .transfer_len = {{ "%u"|format(node["length"]) }}UL,
```

Comment:
Finally is not it too rigid to hard-code the transfer length in the DTS without possibility to adjust it with an API parameter of `sys_dma_start_stream()` par example? 
We might have use cases when we want to transfer different lengths up to a certain maxim (the one we could put in the DTS), might not we ?

Reply:
Yes, length is something specified at dts level by now.

Reply:
this is a protection mechanism to ensure that we can't dma-copy more len that what the shm support, at build time directly